apps/allocations/app/components/App/App.js

```diff
@@ -156,21 +195,21 @@ class App extends React.Component {
               {panel.content && <PanelContent {...panel.data} />}
             </SidePanel>
           </ToastHub>
-        </StyledAragonApp>
+        </Main>
       </Root.Provider>
     )
   }
 }
 
-const StyledAragonApp = styled(AragonApp).attrs({
-  publicUrl: ASSETS_URL,
-})`
-  display: flex;
-  height: 100vh;
-  flex-direction: column;
-  align-items: stretch;
-  justify-content: stretch;
-`
+//const StyledAragonApp = styled(Main).attrs({
```

Comment:
Is this styling going to be used at some point and what's the reasoning behind removing it?